lib/Jit/LLILCJit.cpp

```diff
@@ -290,6 +290,16 @@ CorJitResult LLILCJit::compileMethod(ICorJitInfo *JitInfo,
 
         Passes.add(createRewriteStatepointsForGCPass(false));
         Passes.run(*M);
+
+        // Once PlaceSafepoints phase is run, there will be no references
+        // to the gc.safepointPoll() function. PlaceSafepoints phase inlines 
+        // the contents of gc.safepointPoll() at all polling locations.
+        //
+        // Therefore, remove the function from the Module. 
+        // We'll get this optimization for free once we turn on the 
+        // LLVM inliner.
```

Comment:
Are we sure the checkers will catch if this isn't the case?  If there still is a reference to the safepointPoll?  I'm just wondering if we need an extra assert here.  Thoughts?